### PR TITLE
Update questionnaire response item extensions

### PR DESCRIPTION
**Pull request by vukolov-epam**
Description
----

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/clinical/diagnostics/questionnaire-response.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ The following fields are returned if valued:
 
 * [Choice Answer]
 * [Note]
+* [Modifiable]
```
Comment (sydneanschutz) [inline content/millennium/r4/clinical/diagnostics/questionnaire-response.md:44]: Is Modifiable

File: content/millennium/r4/clinical/diagnostics/questionnaire-response.md [hunk 2]
```diff
@@ -50,6 +51,7 @@ URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/Struct
 -----------------|-------------------------------------------------------------------|-----------------------------------------------
  `choice-answer` | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean)       | Indicates answers come from a list of options.
  `note`          | [`Annotation`](https://hl7.org/fhir/r4/datatypes.html#annotation) | Additional details about a given QuestionnaireResponse group item. Includes author and date/time information.
+ `modifiable`    | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean)       | The extension is present only if activity modifiable is FALSE
```
Comment (sydneanschutz) [inline content/millennium/r4/clinical/diagnostics/questionnaire-response.md:54]: is-modifiable

Comment (sydneanschutz) [inline content/millennium/r4/clinical/diagnostics/questionnaire-response.md:54]: 'Indication of whether data is modifiable or not. Only present when false.'

File: content/millennium/r4/clinical/diagnostics/questionnaire-response.md (hunk 1 above)
Comment (vatnajokull) [reply content/millennium/r4/clinical/diagnostics/questionnaire-response.md]: https://github.com/cerner/fhir.cerner.com/pull/697/commits/9c5a0a983df9702bbae651e4be4907edbd35b377#diff-65d395ab3c58d2ca0d3ac40728ec9e34347612b86c616594927d663c0ccb1c31L44

File: content/millennium/r4/clinical/diagnostics/questionnaire-response.md (hunk 2 above)
Comment (vatnajokull) [reply content/millennium/r4/clinical/diagnostics/questionnaire-response.md]: https://github.com/cerner/fhir.cerner.com/pull/697/commits/9c5a0a983df9702bbae651e4be4907edbd35b377#diff-65d395ab3c58d2ca0d3ac40728ec9e34347612b86c616594927d663c0ccb1c31L54

Comment (vatnajokull) [reply content/millennium/r4/clinical/diagnostics/questionnaire-response.md]: https://github.com/cerner/fhir.cerner.com/pull/697/commits/9c5a0a983df9702bbae651e4be4907edbd35b377#diff-65d395ab3c58d2ca0d3ac40728ec9e34347612b86c616594927d663c0ccb1c31L54

File: content/millennium/r4/foundation/conformance/structure-definition.md [hunk 3]
```diff
@@ -97,6 +97,7 @@ Authorization is not required.
  [`financial-transaction-tendered-amount`]                   | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
  [`financial-transaction-type`]                              | The classification of the transaction.
  [`formatted-text`]                                          | Link to the formatted text for a note. It is used in cases when the note is not a plain text.
+ [`modifiable`]                                              | The extension is present only if activity modifiable is FALSE.
```
Comment (sydneanschutz) [inline content/millennium/r4/foundation/conformance/structure-definition.md:100]: is-modifiable

Comment (sydneanschutz) [reply content/millennium/r4/foundation/conformance/structure-definition.md:100]: line 151 too

Comment (vatnajokull) [reply content/millennium/r4/foundation/conformance/structure-definition.md]: https://github.com/cerner/fhir.cerner.com/pull/697/commits/fd1e5bc0fa24961f4b2451bf238b345d5724f80d#diff-6d3680a1fcdfd40913793f8a19ab5e89c212edcba4a5b88a7ba7cff44c3af047R100

File: content/millennium/r4/clinical/diagnostics/questionnaire-response.md [hunk 4]
```diff
@@ -41,6 +41,7 @@ The following fields are returned if valued:
```
Comment (sydneanschutz) [inline content/millennium/r4/clinical/diagnostics/questionnaire-response.md:41]: once we return this extension QR read and search will no longer be beta, so we can remove the tag from those sections https://github.com/cerner/fhir.cerner.com/blame/fd1e5bc0fa24961f4b2451bf238b345d5724f80d/content/millennium/r4/clinical/diagnostics/questionnaire-response.md#L58

Comment (vatnajokull) [reply content/millennium/r4/clinical/diagnostics/questionnaire-response.md:41]: https://github.com/cerner/fhir.cerner.com/pull/697/commits/ca0df6a51fe1bc45033cea19dcc23f30e78aa3b6#diff-65d395ab3c58d2ca0d3ac40728ec9e34347612b86c616594927d663c0ccb1c31L58

File: content/millennium/r4/foundation/conformance/structure-definition.md [hunk 5]
```diff
@@ -97,6 +97,7 @@ Authorization is not required.
  [`financial-transaction-tendered-amount`]                   | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
  [`financial-transaction-type`]                              | The classification of the transaction.
  [`formatted-text`]                                          | Link to the formatted text for a note. It is used in cases when the note is not a plain text.
+ [`is-modifiable`]                                              | The extension is present only if activity modifiable is FALSE.
```
Comment (MadhurRaj) [inline content/millennium/r4/foundation/conformance/structure-definition.md:100]: lets preserve the alignment here.

File: content/millennium/r4/foundation/conformance/structure-definition.md [hunk 6]
```diff
@@ -147,6 +148,7 @@ Authorization is not required.
 [`financial-transaction-tendered-amount`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount?_format=json
 [`financial-transaction-type`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-type?_format=json
 [`formatted-text`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/formatted-text?_format=json
+[`is-modifiable`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/modifiable?_format=json
```
Comment (MadhurRaj) [inline content/millennium/r4/foundation/conformance/structure-definition.md:151]: is this the right url?

File: content/millennium/r4/foundation/conformance/structure-definition.md (hunk 5 above)
Comment (vatnajokull) [reply content/millennium/r4/foundation/conformance/structure-definition.md]: https://github.com/cerner/fhir.cerner.com/pull/697/commits/bc9407c18cb420c2ad537b1a0c384d77f3647634#diff-6d3680a1fcdfd40913793f8a19ab5e89c212edcba4a5b88a7ba7cff44c3af047R100

File: content/millennium/r4/foundation/conformance/structure-definition.md (hunk 6 above)
Comment (vatnajokull) [reply content/millennium/r4/foundation/conformance/structure-definition.md]: https://github.com/cerner/fhir.cerner.com/pull/697/commits/bc9407c18cb420c2ad537b1a0c384d77f3647634#diff-6d3680a1fcdfd40913793f8a19ab5e89c212edcba4a5b88a7ba7cff44c3af047R151

**Pull request closed without merging**